frontend/src/lib/components/LemonCalendar/LemonCalendar.tsx

```diff
@@ -0,0 +1,141 @@
+        if (props.firstMonth && props.firstMonth !== firstMonth) {
+            setFirstMonth(props.firstMonth)
+        }
+    }, [props.firstMonth])
+
+    return (
+        <div className="LemonCalendar flex items-start gap-4">
+            {range(0, months).map((month) => {
+                const startOfMonth = (firstMonth ? dayjs(firstMonth) : dayjs()).add(month, 'month').startOf('month')
+                const endOfMonth = (firstMonth ? dayjs(firstMonth) : dayjs()).add(month, 'month').endOf('month')
+                const stringMonth = startOfMonth.format('YYYY-MM-DD')
+                // TODO: support the easier US Sunday-first format as well
+                const firstDay = startOfMonth.subtract(startOfMonth.day() === 0 ? 6 : startOfMonth.day() - 1, 'days')
+                const lastDay = endOfMonth.add(endOfMonth.day() === 0 ? 0 : 7 - endOfMonth.day(), 'days')
+                const weeks = props.weeks ?? lastDay.diff(firstDay, 'week') + 1
+                const showLeftMonth = month === 0
+                const showRightMonth = month + 1 === months
+
+                return (
+                    <table className="LemonCalendar__month" key={month}>
+                        <thead>
+                            <tr>
+                                {showLeftMonth && (
+                                    <th>
+                                        <LemonButton
+                                            status="stealth"
+                                            fullWidth
+                                            onClick={() => {
+                                                const newDate = dayjs(firstMonth)
+                                                    .subtract(1, 'month')
+                                                    .format('YYYY-MM-DD')
+                                                setFirstMonth(newDate)
+                                                props.onFirstMonthChanged?.(newDate)
+                                            }}
+                                        >
+                                            <IconChevronLeft />
+                                        </LemonButton>
+                                    </th>
+                                )}
+                                <th colSpan={7 - (showLeftMonth ? 1 : 0) - (showRightMonth ? 1 : 0)}>
+                                    <LemonButton
+                                        status="muted"
+                                        fullWidth
+                                        center
+                                        className="text-xs font-bold text-muted uppercase cursor-default"
+                                    >
+                                        {startOfMonth.format('MMMM')} {startOfMonth.year()}
+                                    </LemonButton>
+                                </th>
+                                {showRightMonth && (
+                                    <th>
```

Comment:
The month name is a bit off-center when there's only a button on one side, it would be a bit nicer if the label were positioned the same regardless of where the buttons are.
<img width="532" alt="Screen Shot 2022-08-31 at 12 59 26" src="https://user-images.githubusercontent.com/4550621/187663801-e88f9c49-d8eb-491a-bd91-12d891f67fab.png">

Reply:
Such positioning is possible if I absolute-position the left/right arrows:

<img width="623" alt="image" src="https://user-images.githubusercontent.com/53387/187685790-d4dfa904-7986-4f12-823a-a80074c17e3f.png">